Nos/Models/VerifiableEvent.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  VerifiableEvent.swift
+//  Nos
+//
+//  Created by Daniel on 24/4/24.
+//
+
```

Comment:
```suggestion
```

We've agreed to not include these generated headers (see #952, which enables this as a check in SwiftLint, so this should have been caught by SwiftLint... 🤔). Could you remove this one?